### ISSUE TITLE
A11y: Self-collapsing mode - Elements must meet min. color contrast ratio thresholds

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5040]
Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds [https://dequeuniversity.com/rules/axe/4.10/color-contrast?application=AxeChrome|https://dequeuniversity.com/rules/axe/4.10/color-contrast?application=AxeChrome|smart-link] [https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html|https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html|smart-link] 

{noformat}<span class="customized-products-upload__dropzone-text-restrictions small">
(Max. 1 file(s), max. 10 MB per file)</span>{noformat}

h3. To solve this problem, you need to fix the following:

Element has insufficient color contrast of 1.71 (foreground color: #bcc1c7, background color: #f7f9fc, font size: 9.2pt (12.25px), font weight: normal). Expected contrast ratio of 4.5:1

!Screen Shot 2024-10-09 at 08.50.22 AM.png|width=914,height=380,alt="Screen Shot 2024-10-09 at 08.50.22 AM.png"!

!Screen Shot 2024-10-09 at 08.51.28 AM.png|width=908,height=378,alt="Screen Shot 2024-10-09 at 08.51.28 AM.png"!

!Screen Shot 2024-10-09 at 08.53.04 AM.png|width=976,height=1236,alt="Screen Shot 2024-10-09 at 08.53.04 AM.png"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **CUS-732**



[created from CUS-732, comment 501895]
